ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/HeaderConverter_Test.java

```diff
@@ -31,9 +33,18 @@
     @Test
     public void testToHeader() throws ParseException {
         final ImmutableList<Request> allRequests = ImmutableList.of(
-                RequestConverter.toRequest(Ds3ModelFixtures.createBucketRequest()),
-                RequestConverter.toRequest(Ds3ModelFixtures.deleteBucketRequest()),
-                RequestConverter.toRequest(Ds3ModelFixtures.getBucketsRequest()));
+                RequestConverter.toRequest(Ds3ModelFixtures.createBucketRequest(),
+            new Ds3DocSpecImpl(
+                ImmutableMap.of(),
+                ImmutableMap.of())),
+                RequestConverter.toRequest(Ds3ModelFixtures.deleteBucketRequest(),
+            new Ds3DocSpecImpl(
+                ImmutableMap.of(),
+                ImmutableMap.of())),
+                RequestConverter.toRequest(Ds3ModelFixtures.getBucketsRequest(),
+            new Ds3DocSpecImpl(
```

Comment:
Can use `Ds3DocSpecEmptyImpl` here. Also, is there any reason your creating 3 doc-specs instead of re-using one?